github/actions_hosted_runners.go

```diff
@@ -0,0 +1,333 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// HostedRunnerPublicIP represents the details of a public IP for github-hosted runner.
+type HostedRunnerPublicIP struct {
+	Enabled bool   `json:"enabled"`
+	Prefix  string `json:"prefix"`
+	Length  int    `json:"length"`
```

Comment:
Could you please add a comment stating what this field represents?
What is it the length of?